### DISCUSSION TITLE
Edit subtitles directly in list view

### DISCUSSION THREAD

**Discussion by Xenos71** [SubtitleEdit/subtitleedit#7093] [Ideas]
Hi,

I believe it would be really helpful and quicker if we had the option to edit subtitles directly in the list view, instead of having to go to the text box every time we need to change or add a sub.

Would it be possible to consider implementing this function?

Thanks in advance,
Xenos

**Comment by oep42** [comment]
Same suggestion [here](https://github.com/SubtitleEdit/subtitleedit/issues/7375).
